app.js

```diff
@@ -13,25 +19,28 @@ app.get(`/api/articles/:article_id`, getArticleById)
 
 app.get(`/api/articles`, getAllArticles)
 
+app.get(`/api/articles/:article_id/comments`, getArticleCommentsById)
+
 
 app.all("/*", (req, res, next) => {
     res.status(404).send({message: "path not found"})
 })
 
 app.use((err, req, res, next) => {
-    if (err.status === 400){
-        console.log(err)
+    if (err.status === 400 && err.message === "invalid id path"){
         res.status(400).send({message: "invalid id path"})
     }
-    if (err.status === 404){
-        console.log(err)
+    next(err)
+})
+
```

Comment:
You could refactor the above and below app.use into a custom error handler (like in th notes https://notes.northcoders.com/courses/js-back-end/error-handling#4-refactoring-to-individual-error-handling-blocks)